ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh

```diff
@@ -63,11 +68,11 @@ compute:
       memoryMiB: 16384
       osDisk:
         diskSizeGiB: 120
-  replicas: 3
+  replicas: ${workers}
 controlPlane:
```

Comment:
I guess we need another section for setting masters' Memory and CPU  as the following
* "compact" - 8 vCPU, 32GB control plane nodes, no workers

when compact was set.